README.md

```diff
@@ -56,23 +48,19 @@ https://annu12340.github.io/Javascript-Notes/
 		
   ### Negative INFINITY
   
-        NEGATIVE_INFINITY is displayed when a number exceeds the lower limit of the floating 
+               NEGATIVE_INFINITY is displayed when a number exceeds the lower limit of the floating 
 		point numbers, which is **-1.797693134862316E+308.**
```

Comment:
``` NEGATIVE_INFINITY is displayed when a number exceeds the lower limit of the floating 
		point numbers, which is **-1.797693134862316E+308.**```

Instead of writing it in 2 lines, pls write it in one line. Here also the bold styling is not rendering

NEGATIVE_INFINITY is displayed when a number exceeds the lower limit of the floating-point numbers, which is        -1.797693134862316E+308